app/presenters/tree_builder_tags.rb

```diff
@@ -44,17 +44,26 @@ def root_options
   end
 
   def x_get_tree_roots(count_only, _options)
+    return @categories.size if count_only
```

Comment:
There a reason the `return` was removed from the "master" version of this PR:  https://github.com/ManageIQ/manageiq-ui-classic/pull/57/files